app/src/androidTest/java/io/intrepid/skeleton/CustomTestRunner.java

```diff
@@ -0,0 +1,13 @@
+package io.intrepid.skeleton;
+
+import android.app.Application;
+import android.content.Context;
+import android.support.test.runner.AndroidJUnitRunner;
+
+public class CustomTestRunner extends AndroidJUnitRunner {
```

Comment:
Creating an AndroidManifest.xml in the androidTest directory doesn't do much since it's the manifest for the test application rather than the application being tested.  We encountered this before at Onshape when we tried to add a test only permission. We would need to create a separate build type for this to work.